Multisig/UI/ClaimToken/TokenDistributionViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  TokenDistributionViewController.swift
+//  Multisig
+//
+//  Created by Mouaz on 9/5/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
```

Comment:
we should update the copyright

Reply:
we have this all over the app, I will change it later with the full rebrand